wiki/en/Running-a-Server.md

```diff
@@ -100,6 +89,10 @@ If you want to run a Server on a Raspberry Pi, you will need to [compile from so
 1. Reload the systemd files `sudo systemctl daemon-reload` and restart the headless Server: `sudo systemctl restart jamulus-headless`
 1. Check all is well with `systemctl status jamulus-headless` (hit `q` to get back to the command prompt).
 
+_To upgrade your Server, just repeat the steps above._
+
+If you are running an unregistered server behind a home internet connection, you might need to enable [port forwarding](port-forwarding) as described below.
+
 ### Configuration
 
 You can control Jamulus with the `systemctl` command. For example, to stop the Server cleanly:
```

Comment:
Lines 105 to 107 aren't _Configuration_.

Reply:
In fact, none of 103 to 139 is giving any guidance on _Configuration_.

Reply:
Could you please give a suggestion? CLI? Administration? Controlling the server?

Reply:
This stuff is currently in a general "Notes" section because I couldn't think of a better name for it. But maybe "Controlling the Server" is good enough.

Reply:
Changed to your suggestion, but I'm still not sure if it's the best way to explain this section. What is it actually about? Somewhat it's related to CLI commands a server admin should know.

Reply:
How about:

Move "You can control Jamulus with the systemctl command...." bit to below the last step of installation.

Move the following lines into the section on Server Types (where it probably better belongs):

_If you are running an unregistered server behind a home internet connection, you might need to enable [port forwarding](http://192.168.0.20:4000/wiki/Running-a-Server#port-forwarding) as described below._

And have the rest read as follows:

---

7. Check all is well with `systemctl status jamulus-headless` (hit `q` to get back to the command prompt).

You can control Jamulus with the systemctl command. For example, to stop the Server cleanly:

`sudo systemctl stop jamulus-headless`

_To upgrade your Server, just repeat the steps above._

## Configuration

### Registered mode

---

... and continue from there.

That will mean the TOC reads as:

- Installation
- Configuration
- Maintenance

Reply:
Can you add your proposal via GitHub's suggestion feature?

Reply:
Hm. All I can work out how to do is edit the file - will that do?

Reply:
Yes. That's ok too: https://github.com/ann0see/jamuluswebsite/edit/feature/minimizeServerManual/wiki/en/Running-a-Server.md

Although a suggestion would be nicer - but not needed.

Reply:
OK I did a commit to that file.

Reply:
How come that commit never turned up?

Reply:
I think it did? Can you check if the thing is edited here

Reply:
Did you use the zip from this PR? I can have a look at the md file again

Reply:
No, I just` git clone` and `git checkout feature/minimizeServerManual`

Reply:
Ok. Try the zip from this PR and check it – maybe I wrongfully reverted your commit

Reply:
The [zip from your repo](https://github.com/ann0see/jamuluswebsite/archive/refs/heads/feature/minimizeServerManual.zip), I assume?  I tried that but it didn't have the changes. I can do them again on https://github.com/ann0see/jamuluswebsite/edit/feature/minimizeServerManual/wiki/en/Running-a-Server.md unless there's some way of using GH to suggest the changes, but I can't see how.

Reply:
OK I re-did it and can see the changes.